testIllegalFieldsDefinition()
     tags: []
 uniqueId: [engine:junit-jupiter]/[class:org.apache.pdfbox.pdmodel.interactive.form.PDAcroFormTest]/[method:testIllegalFieldsDefinition()]
   parent: [engine:junit-jupiter]/[class:org.apache.pdfbox.pdmodel.interactive.form.PDAcroFormTest]
   source: MethodSource [className = 'org.apache.pdfbox.pdmodel.interactive.form.PDAcroFormTest', methodName = 'testIllegalFieldsDefinition', methodParameterTypes = '']
   caught: java.net.UnknownHostException: issues.apache.org
             	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:569)
             	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:284)
             	at java.base/java.net.Socket.connect(Socket.java:668)
             	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:304)
             	at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:181)
             	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:166)
             	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:516)
             	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:604)
             	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:206)
             	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:288)
             	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:193)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:963)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:899)
             	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:179)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1382)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1307)
             	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:223)
             	at java.base/java.net.URL.openStream(URL.java:1264)
             	at org.apache.pdfbox.pdmodel.interactive.form.PDAcroFormTest.testIllegalFieldsDefinition(PDAcroFormTest.java:367)
             	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
             	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
             	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:511)
             	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1450)
             	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2019)
             	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
 duration: 342 ms
   status: ✘ FAILED

testPDFBox3347()
     tags: []
 uniqueId: [engine:junit-jupiter]/[class:org.apache.pdfbox.pdmodel.interactive.form.PDAcroFormTest]/[method:testPDFBox3347()]
   parent: [engine:junit-jupiter]/[class:org.apache.pdfbox.pdmodel.interactive.form.PDAcroFormTest]
   source: MethodSource [className = 'org.apache.pdfbox.pdmodel.interactive.form.PDAcroFormTest', methodName = 'testPDFBox3347', methodParameterTypes = '']
   caught: java.net.UnknownHostException: issues.apache.org
             	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:569)
             	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:284)
             	at java.base/java.net.Socket.connect(Socket.java:668)
             	at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:304)
             	at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:181)
             	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:166)
             	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:516)
             	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:604)
             	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:206)
             	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:288)
             	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:193)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:963)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:899)
             	at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:179)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1382)
             	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1307)
             	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:223)
             	at java.base/java.net.URL.openStream(URL.java:1264)
             	at org.apache.pdfbox.pdmodel.interactive.form.PDAcroFormTest.testPDFBox3347(PDAcroFormTest.java:387)
             	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
             	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
             	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:511)
             	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1450)
             	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2019)
             	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
 duration: 4 ms
   status: ✘ FAILED

--- a/pdfbox/src/test/java/org/apache/pdfbox/pdmodel/interactive/form/PDAcroFormTest.java
+++ b/pdfbox/src/test/java/org/apache/pdfbox/pdmodel/interactive/form/PDAcroFormTest.java
@@ -53,6 +53,7 @@ import org.apache.pdfbox.rendering.TestPDFToImage;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Assertions;
 
 /**
@@ -358,7 +359,7 @@ class PDAcroFormTest
      * @throws IOException
      * @throws URISyntaxException
      */
-    @Test
+    @Test @Disabled
     void testIllegalFieldsDefinition() throws IOException, URISyntaxException
     {
         String sourceUrl = "https://issues.apache.org/jira/secure/attachment/12866226/D1790B.PDF";
@@ -378,7 +379,7 @@ class PDAcroFormTest
      * @throws IOException
      * @throws URISyntaxException 
      */
-    @Test
+    @Test @Disabled
     void testPDFBox3347() throws IOException, URISyntaxException
     {
         String sourceUrl = "https://issues.apache.org/jira/secure/attachment/12968302/KYF%20211%20Best%C3%A4llning%202014.pdf";
